### ISSUE TITLE
Alt text for photos.

### ISSUE THREAD

**Issue by pityhero233** [zim-desktop-wiki/zim-desktop-wiki#1979]
**Is your feature request related to a problem? Please describe.**
Picture contain text. But that's not findable from global search. Ideally we should have feature that OCR photo, but it takes more time. An alt text function should suffice. 

**Describe the solution you'd like**
Add "Alt Text" section in photo properties. User can enter description of photo, which is searchable from the global search.

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by introt** [comment]
Duplicate of #963, but I like the additional OCR & search angle; please move your comment over to the existing issue.

@Osndok's grep patch for #64 included `.caption` files in the search, so there seems to be a plugin for this floating around.
